### ISSUE TITLE
OBS Fails to properly launch if this filter isn't removed before closing OBS

### ISSUE THREAD

**Issue by jrwagz** [exeldro/obs-virtual-cam-filter#13]
I installed this extension, and then launched OBS and applied the new "Virtual Camera" filter to a scene, and turned off the old "VirtuCam" filter (or should I say made it not-visible) on the same scene.  The next time I tried to launch OBS it fails to even open up the OBS GUI at all.

My assumption is that this filter is trying to start, but is failing for some reason, which halts OBS from opening.

Are there any logs I can look at to get more details?
How do I uninstall this extension?

Any help is appreciated.

**Comment by jrwagz** [comment]
Upon further investigation, I did the following:

- removed old VirtuCam extension/plugin (deleting dll file)
- removed this plugin (deleting dll file)

After that I was able to get OBS to launch again.  And go in and remove the leftover filter references to uninstalled plugins.

Then I reinstalled this plugin, and applied it to a scene, and it worked.

I then restarted OBS, and it wouldn't launch.

I had to then remove the extension again, and remove the old filter reference.

Then I installed it again, and applied it to a scene, and so long as I remove the filter before closing OBS, then OBS seems to launch fine, but if I don't remove it, OBS won't ever launch.
